interview-questions.md

```diff
@@ -6,52 +6,416 @@ First, without external resources. Challenge yourself to answer from memory as i
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-1. What is the difference between a parameter and an argument?
+-------
 
-Your answer:
+### 1. What is the difference between a parameter and an argument?
+
+Your answer: 
+
+A parameter is a variable declared in a function's definition, while an argument is the actual value passed to the function when it is called.
 
 Researched answer:
 
```

Comment:
Superb way to show that you can research, comprehend information, and share that information.